sage-ios/SAGE/SAGE/AddSchoolLocationTableViewController.swift

```diff
@@ -0,0 +1,102 @@
+//
```

Comment:
sadly no not an easy way :( 
you were probably right about uitableviewcontroller, we could change it to a regular uiviewcontroller and then make the search bar fixed if you want